addon/components/swiper-container.js

```diff
@@ -227,9 +242,12 @@ export default Component.extend({
 
       if (this.get('loop')) {
         let swiper = this.get('_swiper');
-        index = $(swiper.slides)
-          .filter(`[data-swiper-slide-index=${this.get('currentSlide')}]`)
-          .prevAll().length;
+        Object.keys(swiper.slides).forEach((key) => {
```

Comment:
I might considering adding a new computed attribute:
```js
slides: computed('_swiper.slides.[]', function() {
  return [].slice.call(this.get('_swiper.slides'))
})
```
So you can dry up this code a little and not need to lookup each slide with each itteration of the forEach.

Reply:
This isn't a huge issue but using a forEach here is less efficient than using a filter.

Reply:
Gonna give it a try